### PR TITLE
Add installation instructions to readme

### PR DESCRIPTION
**Pull request by owenpearson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,17 @@ Visit the [Ably Asset Tracking](https://ably.com/documentation/asset-tracking) d
 - [Introducing Ably Asset Tracking - public beta now available](https://ably.com/blog/ably-asset-tracking-beta)
 - [Accurate Delivery Tracking with Navigation SDK + Ably Realtime Network](https://www.mapbox.com/blog/accurate-delivery-tracking)
 
+## Installation
+
+To use ably asset tracking in your app, install it as a dependency:
```
Comment (kavalerov) [inline README.md:34]: This should be "Ably Asset Tracking" (capitalised first letters)

**Review by kavalerov — CHANGES_REQUESTED** [review]
Minor spelling issue, otherwise I think this is good to go (for the scope of this PR). Separately I would want to look if we want to provide any additional information to the README later

File: README.md (hunk 1 above)
Comment (owenpearson) [reply README.md]: Good point, fixed in 2c522e9

**Comment by owenpearson** [comment]
@kavalerov I opened https://github.com/ably/ably-asset-tracking-js/issues/55 so this doesn't get lost in the ether.

**Review by kavalerov — APPROVED** [review]
LGTM, thanks!

**Pull request merged**